src/tests/unit/isCohortDuplicate.test.js

```diff
@@ -0,0 +1,95 @@
+import { isCohortDuplicate } from "../../pages/AdminForm";
+
+describe("Duplicate cohort submittion check", () => {
+  it("returns an array with lenght 1 for duplicate cohort", () => {
+    const isCohortDuplicate = () => {
+      const props = {
+        cohortName: "cohort 10",
+        cohortType: "backend",
+        existingCohorts: [{ cohortName: "cohort 10", cohortType: "backend" }]
+      };
+
+      const { cohortName, cohortType, existingCohorts } = props;
+      return existingCohorts
+        .filter(cohort => cohort.cohortType === cohortType)
+        .filter(
+          cohort => cohort.cohortName.toLowerCase() === cohortName.toLowerCase()
+        ).length;
```

Comment:
Note from Marissa about my unit test - "It's best that you import the function you're trying to test not rewrite it. One of the good things about unit tests is that it documents how the function works so if someone changes a critical piece of the functionality the test will fail and they will need to think about whether they are changing something they shouldn't be". Try to define the helper function outside of the component and then export it and import it to this test.

Reply:
well I dont understand if I write my function again in helper function then what about this method in AdminForm? do I have to import again from helper in AdminForm?

Reply:
yeah she suggested we rewrite it so it is reusuable, and keep it in a file in a folder called helpers and import it into both the admin form and the test. Or if you keep it in the admin form, just write it above the component function and call it with arguments within the component function